{\rtf1\ansi\ansicpg1252\cocoartf2822
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fnil\fcharset0 AppleColorEmoji;}
{\colortbl;\red255\green255\blue255;}
{\*\expandedcolortbl;;}
\paperw11900\paperh16840\margl1440\margr1440\vieww11520\viewh8400\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0

\f0\fs24 \cf0 diff --git a/index.html b/index.html\
new file mode 100644\
index 0000000..b6fc4c3\
--- /dev/null\
+++ b/index.html\
@@ -0,0 +1,7 @@\
+<!DOCTYPE html>\
+<html lang="de">\
+<head>\
+  <meta charset="UTF-8">\
+  <title>Hygia Loft Autopilot Test</title>\
+</head>\
+<body><h1>
\f1 \uc0\u55357 \u56960 
\f0  Autopilot funktioniert!</h1></body>\
+</html>}